README.md

```diff
@@ -40,15 +52,91 @@ or
 python setup.py develop
 ```
 
```

Comment:
```suggestion
Then, run tests:

pytest
```

Reply:
Please add code-block syntax ` ```py ...```  `around `pytest`. I couldn't add it here.

Reply:
Because there is no any environment, it is better to add `python3 -m` for both `s2spy` and `pytest`. Otherwise, older versions of python will be picked by `pytest`.

Reply:
Thanks for the suggestion! I will add these to the readme.